worker/src/models/property/PropertyModel.ts

```
@@ -1,98 +1,165 @@
+          lte: longitude + lonDiff,
+        },
+      },
+      include: {
+        userProfile: true,
+      },
+      orderBy: {
+        createdAt: 'desc',
+      },
+    });
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)